### PR TITLE
stumpwm: Added newer version of stumpwm from its git repository (alon…

### PR DESCRIPTION
**Pull request by fstamour**
###### Motivation for this change

Just wanted a newer version of stumpwm.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/window-managers/stumpwm/default.nix [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
+    "latest" = {
+      name = "0.9.9";
+      rev = "refs/tags/0.9.9";
+      sha256 = "05fkng2wlmhy3kb9zhrrv9zpa16g2p91p5y0wvmwkppy04cw04ps";
+      patches = [ ./fix-module-path.patch ];
+    };
+    "git" = {
+        name = "git-20160617";
+        rev = "7d5b5eb76aa656baf5a8713f514937765f66b10a";
+	sha256 = "1b7lr9rj29qrazzx4xwr69fw6a89pnkycpy9jn81k84k0v22n1q0";
```
Comment (aristidb) [inline pkgs/applications/window-managers/stumpwm/default.nix]: nox-review seems to complain about the hash for this?

**Comment by mpsyco** [comment]
I was unable to reproduce the problem, I tried `nox-review pr 16310` and `nix-shell -p nox --run "nox-review pr 16310"` and they both worked.
I ran `nix-collect-garbage` and _then_ only I could reproduce the problem.

Any idea on what the problem was on my setup?

Anyhow. I will push the fix  in a minute.

**Comment by the-kenny** [comment]
I think `refs/tags/0.9.9` doesn't work. Running `% nix-prefetch-git https://github.com/stumpwm/stumpwm refs/tags/0.9.9` yields `0hmvbdk2yr5wrkiwn9dfzf65s4xc2qifj0sn6w2mghzp96cph79k` for me now.

**Comment by mpsyco** [comment]
I didn't touched the  `refs/tags/0.9.9` per se, that was already in the master branch. But from the diff I can see that I have inadvertently changed the sha512 for the version 0.9.9 (but it still works on my setup, which is incredibly annoying).

@the-kenny TL;DR No, the `refs/tags/0.9.9` works perfectly, I just messed up the sha256.

**Comment by mpsyco** [comment]
It should be all right now.

**Comment by the-kenny** [comment]
Merged with a minimal improvement in `all-packages.nix` in 5e7bcc41b34657cd89e586d464862e04da43fda5

**Pull request closed without merging**